UniversalInstaller/Mac/MacInstall.cs

```diff
@@ -69,9 +69,13 @@ public bool Install()
             ProcessHandler.Run("chmod", "755 " + Path.Combine(GetLocation(), "OSX-FOG-TRAY.app"));
             ProcessHandler.Run("chmod", "+x " + Path.Combine(GetLocation(), "OSX-FOG-TRAY.app/Contents/MacOS/OSX-FOG-Tray"));
```

Comment:
As I can't comment on a file commit that has no viewable code change (like binary files) I will comment on  UniversalInstaller/Scripts/OSX-FOG-TRAY.zip here. Seems like you only removed that empty file. Did you also want to add the new ZIP file? On the other hand I still have not found the time to look into how to create the OSX-FOG-TRAY.zip file contents. Maybe we don't even want to add this as a binary file?!

Reply:
It complained at me when trying to add it; UniversalInstaller/Scripts/OSXFOGTray.zip is actually specifically listed in .gitignore (https://github.com/FOGProject/fog-client/blob/6ab512601194aaf953430db693a54f61b4e95733/.gitignore#L5).  Once the process for creating Fog Tray zip is worked out, then I would certainly be all for ditching the binary

Reply:
and it would be even better if I were posting under the correct account.....

Reply:
Going to force it into my next commit.